docs/mkdocs.yml

```diff
@@ -83,6 +83,7 @@ nav:
             -   Internal:
                     - projects/internal/Internal.md
                     - projects/internal/BuildMetrics.md
+                    - projects/internal/CriticalPath.md
```

Comment:
is it really internal? plugin looks like reusable

Reply:
I use it to express the status of the plugin and our plans to support it.
Ideally, we should have a more formal contract for this, like experimental/alpha/beta/... in Kotlin. Now it's not clear what we guarantee.